terraform/create_backend_vars.sh

```diff
@@ -11,9 +11,12 @@ fi
 ACCESS_KEY=$(echo "${S3_CREDENTIALS}" | jq -r '.access_key_id')
 SECRET_KEY=$(echo "${S3_CREDENTIALS}" | jq -r '.secret_access_key')
 REGION=$(echo "${S3_CREDENTIALS}" | jq -r '.region')
+BUCKET=$(echo "${S3_CREDENTIALS}" | jq -r '.bucket')
 
 {
   echo "access_key = \"$ACCESS_KEY\""
   echo "secret_key = \"$SECRET_KEY\""
   echo "region = \"$REGION\""
+  echo "bucket = \"$BUCKET\""
+  echo "prefix = \"$BUCKET\""
```

Comment:
This is an issue, it shouldn't break anything since we never even run this file except as testing steps. I didn't use them through out work on this branch

Reply:
Not sure what this should be in the limited  environments we expect we would ever run these in. Probably "dev" or "local"